networkx/generators/classic.py

```diff
@@ -250,6 +250,11 @@ def complete_graph(n, create_using=None):
     A complete graph on `n` nodes means that all pairs
     of distinct nodes have an edge connecting them.
 
+    .. plot::
+        A plotting example:
+        >>> import networkx as nx
+        >>> nx.draw(nx.complete_graph(5))
+
```

Comment:
Hmm, not sure if this one works. https://output.circle-artifacts.com/output/job/d1b0bd96-8060-4bb7-9095-f1980fdf6db4/artifacts/0/doc/build/html/reference/generated/networkx.generators.classic.complete_graph.html#networkx.generators.classic.complete_graph doesn't show anything. @rossbar is there something specific to set up here to turn this on?

Reply:
Yes, I think there's maybe another extension missing in conf.py